conda/resolve.py

```diff
@@ -7,8 +7,8 @@
 import copy
 from logging import DEBUG, getLogger
 
-from .auxlib.collection import frozendict
 from .auxlib.decorators import memoize, memoizemethod
+from ._vendor.frozendict import frozendict
```

Comment:
```suggestion
from ._vendor.frozendict import FrozenOrderedDict as frozendict
```

Maybe that would be enough to make this possible?